### ISSUE TITLE
Classes in Iceberg-Libgit-Tonel should be properly tagged

### ISSUE THREAD

**Issue by astares** [pharo-vcs/iceberg#1090]
IceLibgitTonelFileUtils
IceLibgitTonelLog
IceLibgitTonelReader
IceLibgitTonelLogReader
IceLibgitTonelWriter

are "Uncategorized" in latest Iceberg and should be properly tagged

**Comment by estebanlm** [comment]
no, they don't.
all of that are in package "Iceberg-Libgit-Tonel".
where did you spotted this?

they do not have a tag, but that's correct (not all classes need to be tagged inside a package, just when that makes sense)

**Comment by astares** [comment]
@estebanlm They will be viewed as "Uncategorized" and to be honest I really would give them all a tag within a package for consistency. 

![image](https://user-images.githubusercontent.com/5980033/48499471-d2f6c380-e838-11e8-9436-3e1295da55aa.png)

**Comment by astares** [comment]
Typically if you have no other idea just use the "Base" tag.

**Comment by estebanlm** [comment]
yes, I still disagree all manifests has to be in `Manifest` tag and then all classes needs to have a package tag.  It seems like a lot of bureaucracy for small packages, isn't? (while of course this is very valuable in large packages).

**Comment by astares** [comment]
@estebanlm you may call this bureaucracy, I would call this disciplined development, clean code and consistency over all packages!

If we cant work professionally we should skip our efforts on Pharo ...
